cmd/aries-agent-mobile/pkg/wrappers/command/aries.go

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Following the existing architecture, this package belongs under `pkg/controller`

Reply:
do you mean i should move `wrappers` into the main aries pkg or i should instead have cmd/aries-agent-mobile/pkg/wrappers/**_controller_**/command/aries.go

Reply:
@ToJen 

> do you mean i should move `wrappers` into the main aries pkg or i should instead have cmd/aries-agent-mobile/pkg/wrappers/**_controller_**/command/aries.go

Move them under the existing `pkg/controller` folder. Something like this: `cmd/aries-agent-mobile/pkg/wrappers/command` -> `pkg/controller/mobile`